api/namespace/applications/helpers/helpers.py

```diff
@@ -21,6 +21,7 @@ def get_blank_forms(fund_id: str, round_id: str):
         """
         fund = get_fund(fund_id)
         fund_round = get_round(fund_id, round_id)
+        # Check the fund and round provided exist in the fund store
         if fund and fund_round:
```

Comment:
@SentientHousePlant - not sure what we are running this check for. Is the comment I added here accurate?